src/instructlab/lab.py

```diff
@@ -292,8 +292,14 @@ def diff(ctx, taxonomy_path, taxonomy_base, yaml_rules, quiet):
     type=click.INT,
     help="The context size is the maximum number of tokens considered by the model, for both the prompt and response. Defaults to 4096.",
 )
+@click.option(
+    "--model-family",
+    type=str,
+    default="merlinite",
+    help="Model family is used to specify which chat template to serve with",
+)
```

Comment:
I'm really confused, why are chatting/serving with different model families? 
Why do we need a new flag? 
@oindrillac maybe you can help clarify? I was under the assumption that this PR will only have changes to `generate` based off of the last conversation.

Reply:
`ilab generate` generates based on the model that's served by `ilab serve`. The model that's served via `ilab serve` is downloaded via `ilab download`. It's absolutely in scope to have `download`, `serve`, and `generate` parameterized to use a different model for `generate`.